src/keylessBackup/KeylessBackupPhoneCodeInput.test.tsx

```diff
@@ -129,4 +131,49 @@ describe('KeylessBackupPhoneCodeInput', () => {
     jest.runOnlyPendingTimers()
     expect(navigate).not.toHaveBeenCalled()
   })
+  it('displays the help bottom sheet and track analytics event when pressing the help button', async () => {
+    const { getByTestId } = renderComponent()
+
+    expect(getByTestId('KeylessBackupPhoneCodeInputHelp')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackupPhoneCodeInputHelp'))
+    expect(getByTestId('KeylessBackupPhoneCodeInput/HelpInfoBottomSheet')).toBeTruthy()
+    expect(ValoraAnalytics.track).toHaveBeenCalledWith(
+      KeylessBackupEvents.cab_phone_verification_help
+    )
+  })
+  it('dismisses help bottom sheet and track analytics event when pressing go back button', async () => {
```

Comment:
technicaly doesn't test that the bottom sheet is dismissed, just that the analytics event is fired